src/controllers/colormapperjsproxy.cpp

```diff
@@ -38,8 +38,9 @@ QScriptValue ColorMapperJSProxyConstructor(QScriptContext* pScriptContext, QScri
     QScriptValueIterator it(argument);
     while (it.hasNext()) {
         it.next();
-        QColor color(it.name());
-        if (color.isValid()) {
+        bool isInt = false;
+        QColor color(it.name().toInt(&isInt));
+        if (isInt && color.isValid()) {
```

Comment:
Do we want some additional validation here (check if int is in range 0 to 0xFFFFFF)?

Reply:
Does QColor::isValid already do that?

Reply:
Noz sure. QColor does allow an alpha channel so at least values up to 0xFFFFFFFF will be valid.

Reply:
Okay, shall we merge this as is?

Reply:
ping